client/ayon_core/hosts/traypublisher/addon.py

```diff
@@ -57,3 +58,62 @@ def launch():
     from ayon_core.tools import traypublisher
 
     traypublisher.main()
+
+
+@cli_main.command()
```

Comment:
I would simplify the options
- `--csv-filepath` > `--filepath`
- `--project-name` > `--project`
- `--asset-name` > `--folder`
- `--task-name` > `--task`

Reply:
done in 638c68324139fbe365d36615ef21910a83f2fab2